.github/workflows/e2e.yml

```diff
@@ -23,14 +23,36 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version: ${{ env.GO_VERSION }}
-      - name: Setup kind
-        uses: engineerd/setup-kind@v0.5.0
+      # - name: Setup kind
```

Comment:
will it be provided by default?

Reply:
The kind binary will be installed using e2e_setup.sh (see: https://github.com/openshift-online/maestro/blob/main/test/e2e/setup/e2e_setup.sh#L20-L25)).
This action sets up a KinD cluster named **kind**, which is not needed for our purposes. Instead, we will create a KinD cluster named **maestro** with our own configuration.